Last Batch Policy Changes for remaining readers

swetha097 opened the pull request.

fiona-gladwin committed:
Minor changes shard_count argument name

fiona-gladwin committed:
Rename set and get functions of data_info to decoded_data_info

fiona-gladwin committed:
Merge branch 'swbs/audio/pr1' into swbs/audio/pr2

Commit:
Fix shard_size and audio source evaluation

Commit:
Changes in file_source_reader - to minimize the I/O operations

Commit:
Changes in the variable name

Commit:
Changes in the variable names of the audio source evalution

SundarRajan28 committed:
Merge branch 'swbs/audio/pr3' into audio_pr4

swetha097 committed:
Use set instead of vector

swetha097 committed:
Minor bug fixes

fiona-gladwin committed:
Minor fixes

Remove changes to update the
filenames vector incase of padding
Fix the pipeline when pad is off and use idx in a continuous manner

fiona-gladwin committed:
Fix drop policy without padding

To skip and batch and start with the previously padded idx in last batch

fiona-gladwin committed:
Fix pytorch iterator - PARTIAL policy

fiona-gladwin committed:
Merge branch 'swbs/audio/pr10' of https://github.com/swetha097/rocAL into lbp_fix_pr10

SundarRajan28 committed:
Merge branch 'audio_pr4' into swbs/audio/pr5

SundarRajan28 committed:
Merge branch 'swbs/audio/pr5' into swbs/audio/pr6

SundarRajan28 committed:
Merge branch 'swbs/audio/pr6' into swbs/audio/pr8

SundarRajan28 committed:
Merge branch 'swbs/audio/pr8' into swbs/audio/pr9

SundarRajan28 committed:
Merge branch 'swbs/audio/pr9' into swbs/audio/pr7

SundarRajan28 committed:
Merge remote-tracking branch 'upstream/develop' into swbs/audio/pr2

SundarRajan28 committed:
Merge branch 'swbs/audio/pr2' into swbs/audio/pr3

SundarRajan28 committed:
Merge branch 'swbs/audio/pr3' into audio_pr4

SundarRajan28 committed:
Merge branch 'audio_pr4' into swbs/audio/pr5

SundarRajan28 committed:
Merge branch 'swbs/audio/pr5' into swbs/audio/pr6

SundarRajan28 committed:
Merge branch 'swbs/audio/pr6' into swbs/audio/pr8

SundarRajan28 committed:
Merge branch 'swbs/audio/pr8' into swbs/audio/pr9

SundarRajan28 committed:
Merge branch 'swbs/audio/pr9' into swbs/audio/pr7

fiona-gladwin committed:
Merge branch 'develop' of https://github.com/ROCm/rocAL into swbs/audio/pr1

fiona-gladwin committed:
Merge branch 'swbs/audio/pr1' into swbs/audio/pr2

fiona-gladwin committed:
Revert empty line removed in CMakeLists.txt

fiona-gladwin committed:
Removed prefix original for audio vectors

fiona-gladwin committed:
Fix PARTIAL

Fix issue with batch size greater than dataset size

swetha097 committed:
Reduce overall time for audio source evalution

swetha097 committed:
Fix shard_size and stick to shard issue seen with convergence

swetha097 committed:
Resolve PR comments

swetha097 committed:
Add @params to all args in pytorch.py

swetha097 committed:
Fix build issue

fiona-gladwin committed:
Merge branch 'develop' of https://github.com/ROCm/rocAL into swbs/audio/pr2

fiona-gladwin committed:
Merge branch 'swbs/audio/pr7' of https://github.com/swetha097/rocAL into swbs/audio/pr10

swetha097 committed:
Minor changes in unit test

swetha097 committed:
Merge branch 'swbs/audio/pr2' of https://github.com/swetha097/rocAL into swbs/audio/pr2

swetha097 committed:
Minor changes

swetha097 committed:
Change ROCAL instaces to rocAL in pytorch.py

fiona-gladwin committed:
Merge branch 'swbs/audio/pr2' into swbs/audio/pr3

swetha097 committed:
Resolve the PR comments

swetha097 committed:
Merge branch 'swbs/audio/pr3' of https://github.com/swetha097/rocAL into swbs/audio/pr3

swetha097 committed:
Minor changes in decoders.py - Modify the comment for shard_size

fiona-gladwin committed:
Merge branch 'swbs/audio/pr2' of https://github.com/swetha097/rocAL into swbs/audio/pr3

fiona-gladwin committed:
Merge branch 'swbs/audio/pr3' of https://github.com/swetha097/rocAL into swbs/audio/pr3

fiona-gladwin committed:
Merge branch 'develop' of https://github.com/ROCm/rocAL into swbs/audio/pr3

swetha097 committed:
Merge branch 'swbs/audio/pr10_training' into swbs/audio/pr10

swetha097 committed:
Fix shard_size

SundarRajan28 committed:
Merge branch 'swbs/audio/pr3' into audio_pr4

swetha097 committed:
Minor changes

swetha097 committed:
Changes in pipeline.py and decoders.py

swetha097 committed:
Merge remote-tracking branch 'origin/swbs/audio/pr10_training' into swbs/audio/pr10

swetha097 committed:
Address the PR comments

swetha097 committed:
Address Review comments

swetha097 committed:
Remove print statement

swetha097 committed:
Merge branch 'swbs/audio/pr10' into lbp_fix_pr10

swetha097 committed:
Fix the count_items

swetha097 committed:
Make Sharding similar to DALI

swetha097 committed:
Fix issues with DROP policy by introducing a new vector for padding

swetha097 committed:
Minor fixes

swetha097 committed:
Comment out print statements

swetha097 committed:
Add changes for shard_size LBP testing

swetha097 committed:
Fix DROP Policy with shard_size > 0

swetha097 committed:
Fix Stick_to_Shard=False

swetha097 committed:
Fix PARTIAL policy and code clean up

swetha097 committed:
fix last_batch_padded size when shard_size > 0

swetha097 committed:
Fix Drop policy - we skip the dropped batch in the next epoch

swetha097 committed:
Fix single shard outputs

swetha097 committed:
Remove the commented code and fix the padding code in open()

swetha097 committed:
Remove div by num_shards in decoders.py

fiona-gladwin committed:
Introduce Audio layouts

fiona-gladwin committed:
Add layout changes for spectrogram

swetha097 committed:
Fix the unit tests - c++ & python

fiona-gladwin committed:
Merge branch 'swbs/audio/pr5' of https://github.com/swetha097/rocAL into swbs/audio/pr5_layout

swetha097 committed:
Code clean up and formatting

swetha097 committed:
Minor code clean up

swetha097 committed:
code clean up in pytorch.py

fiona-gladwin committed:
Add layout changes for spectrogram

fiona-gladwin committed:
Pass layouts for MelFilterBank

fiona-gladwin committed:
Fix ToDecibels

Pass layouts for ToDecibels

fiona-gladwin committed:
Fix Normalize

fiona-gladwin committed:
Fix build issue

fiona-gladwin committed:
Merge branch 'swbs/audio/pr5_layout' of https://github.com/swetha097/rocAL into swbs/audio/pr7_layout

fiona-gladwin committed:
Fix python unit test

swetha097 committed:
Merge remote-tracking branch 'swe_fork/swbs/audio/pr7_layout' into swbs/lbp_fixes_pr10

fiona-gladwin committed:
Minor fix

swetha097 committed:
Pass LBP to decoders instead of the Pipeline creation

Commit:
Merge branch 'swbs/lbp_fixes_pr10_pass_readers' into swbs/lbp_fixes_pr10

swetha097 committed:
Update pipeline.py - Remove commented code

swetha097 committed:
Update pipeline.py - Remove commented out code

SundarRajan28 committed:
Adding changes for spec layout changes

SundarRajan28 committed:
Merge branch 'swbs/audio/pr6' into swbs/audio/pr8

SundarRajan28 committed:
Merge branch 'swbs/audio/pr8' into swbs/audio/pr7

SundarRajan28 committed:
Adding changes to MFB and normalize nodes

swetha097 committed:
Update node_slice.cpp

swetha097 committed:
Update node_slice.h

swetha097 committed:
Resolve PR comments

fiona-gladwin committed:
Merge branch 'swbs/audio/pr5_layout' into swbs/audio/pr5

fiona-gladwin committed:
Merge branch 'swbs/audio/pr5' into swbs/audio/pr6

fiona-gladwin committed:
Merge branch 'swbs/audio/pr6' into swbs/audio/pr8

fiona-gladwin committed:
Merge branch 'swbs/audio/pr8' into swbs/audio/pr9

fiona-gladwin committed:
Merge branch 'swbs/audio/pr9' into swbs/audio/pr7

swetha097 committed:
Merge remote-tracking branch 'open_source/develop' into swbs/audio/pr3

swetha097 committed:
Merge branch 'swbs/audio/pr3' into audio_pr4

swetha097 committed:
Merge branch 'audio_pr4' into swbs/audio/pr5

swetha097 committed:
Merge remote-tracking branch 'origin/swbs/audio/pr5' into HEAD

swetha097 committed:
Merge branch 'temp_swbs/audio/pr6' into swbs/audio/pr6

swetha097 committed:
Merge branch 'swbs/audio/pr6' into swbs/audio/pr8

swetha097 committed:
Merge branch 'swbs/audio/pr8' into swbs/audio/pr9

swetha097 committed:
Merge branch 'swbs/audio/pr9' into swbs/audio/pr7

swetha097 committed:
Merge branch 'swbs/audio/pr7' into swbs/lbp_fixes_pr10

swetha097 committed:
Fix downmix failing case and resolve the issue with merge

swetha097 committed:
Fix issue with file_source_reader.cpp when file_list is not used

swetha097 committed:
Resolve PR comments - Sundar

swetha097 committed:
Fix file_source_reader.cpp

SundarRajan98 committed:
Fix shuffle issues

SundarRajan28 committed:
Merge remote-tracking branch 'upstream/develop' into swbs/audio/pr7

sbavasab committed:
Adding comments to all if conditions

swetha097 committed:
Add Cmake changes for HIP backend

swetha097 committed:
Add LPB & Sharding changes for cifar10 reader

swetha097 committed:
Add LBP & Sharding changes to coco-file-source-reader

swetha097 committed:
Add LBP & Sharding changes to TF Record reader

swetha097 committed:
Adding LBP & Sharding changes in caffe lmdb reader

swetha097 committed:
Add LBP and sharding changes in caffe2 lmdb reader

swetha097 committed:
Add support for LBP & Sharding in mxnetIO reader

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' into swbs/lbp_readers_pr11

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' into swbs/lbp_readers_pr11

SundarRajan28 committed:
Merge remote-tracking branch 'upstream/develop' into swbs/audio/pr7

swetha097 committed:
Merge remote-tracking branch 'open_source/develop' into swbs/audio/pr9

SundarRajan28 committed:
Fix merge conflicts

SundarRajan28 committed:
Merge remote-tracking branch 'upstream/develop' into swbs/audio/pr7

swetha097 committed:
Merge remote-tracking branch 'open_source/develop' into develop

swetha097 committed:
Merge remote-tracking branch 'open_source/develop' into swbs/audio/pr9

SundarRajan28 committed:
Merge remote-tracking branch 'upstream/develop' into swbs/audio/pr7

SundarRajan28 committed:
Resolving review comments

swetha097 committed:
Merge branch 'develop' into swbs/audio/pr9

SundarRajan28 committed:
Merge remote-tracking branch 'upstream/develop' into swbs/audio/pr9

SundarRajan28 committed:
Merge branch 'swbs/audio/pr9' into swbs/audio/pr7

SundarRajan28 committed:
Merge remote-tracking branch 'upstream/develop' into swbs/audio/pr7

swetha097 committed:
Merge remote-tracking branch 'swe_fork/swbs/audio/pr7' into swbs/lbp_fixes_pr10

swetha097 committed:
Fix a minor warning in file source reader

LakshmiKumar23 committed:
Merge branch 'develop' into swbs/audio/pr7

SundarRajan28 committed:
Resolving review comments

swetha097 committed:
LBP comments resolution

LakshmiKumar23 committed:
Merge branch 'develop' into swbs/audio/pr7

SundarRajan28 committed:
Resolving review comments

SundarRajan28 committed:
Merge remote-tracking branch 'upstream/develop' into swbs/lbp_fixes_pr10

SundarRajan28 committed:
Merge branch 'swbs/audio/pr7' into swbs/lbp_fixes_pr10

SundarRajan28 committed:
Formatting changes

swetha097 committed:
Resolving Final Set of PR comments

swetha097 committed:
Combine with OR condition

swetha097 committed:
Remove the pad_last_batch_repeated print statement from decoders.py

fiona-gladwin committed:
Merge branch 'develop' of https://github.com/ROCm/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Add shard_size and stick_to_shard variables in args

swetha097 committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

swetha097 committed:
Minor spelling fix

swetha097 committed:
Merge remote-tracking branch 'swe_fork/swbs/lbp_fixes_pr10' into swbs/lbp_readers_pr11

swetha097 committed:
Merge remote-tracking branch 'swe_fork/swbs/lbp_fixes_pr10' into swbs/lbp_readers_pr11

swetha097 committed:
Commit for resolving PR comments

swetha097 committed:
Commit for resolving PR comments

swetha097 committed:
Merge branch 'swbs/lbp_readers_pr11' of https://github.com/swetha097/rocAL into swbs/lbp_readers_pr11

swetha097 committed:
Update cifar10_data_reader.cpp

Minor build fix

swetha097 committed:
Update mxnet_recordio_reader.cpp

swetha097 committed:
Update cifar10_data_reader.cpp

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

swetha097 committed:
Make changes to insert the padded data in the file_names vector

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Support to pass the variables fo lbp as struct

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

swetha097 committed:
Fix segmentation fault

SundarRajan28 committed:
Merge branch 'ROCm:develop' into swbs/lbp_fixes_pr10_pr_comments

swetha097 committed:
Merge remote-tracking branch 'origin/swbs/lbp_fixes_pr10_pr_comments' into swbs/lbp_readers_pr11

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10_pr_comments' into swbs/lbp_fixes_pr10

swetha097 committed:
Resolve PR comments

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Resolve PR comments

swetha097 committed:
Resolve PR comments

swetha097 committed:
Use PreComputed start and end indices

swetha097 committed:
Use precomputed shard_idx start and end in initialize

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

swetha097 committed:
Merge remote-tracking branch 'origin/swbs/lbp_fixes_pr10_pr_comments' into swbs/lbp_fixes_pr10

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

swetha097 committed:
Initialize the Sharding info using ShardingInfo()

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
convert the signed to int32_t type

swetha097 committed:
temp commit for struct changes

Commit:
Fix the struct changes - All the test cases passing

Commit:
Remove any print statements

Commit:
Add support to Pass the decode size policy from the user

Commit:
Add support to Pass the decode size policy from the user

swetha097 committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Rename RocalShardingInfo for ShardingInfo and vice-versa

swetha097 committed:
xywh roi copy

swetha097 committed:
Fix decoders.py for image decoders

swetha097 committed:
Make stick_to_shard True by default

fgladwin committed:
Merge branch 'develop' of https://github.com/ROCmSoftwarePlatform/rocAL into swbs/lbp_fixes_pr10

swetha097 committed:
Minor changes to the copy_data function

fgladwin committed:
Rename to x_offset and y_offset in copy_data

fgladwin committed:
Minor changes - remove unused variables

fgladwin committed:
Minor change - Variable names

fgladwin committed:
Update Doxygen comments and comments of API

fgladwin committed:
Merge branch 'swbs/lbp_fixes_pr10' of https://github.com/swetha097/rocAL into swbs/lbp_fixes_pr10

LakshmiKumar23 committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

Commit:
Make the rocalShardingInfo as the last param for Audio loaders

Commit:
Remove unused variables and functions in file_source_reader.cpp & .h files

Commit:
Remove the doctring explanation for unused params

Commit:
Change the explanation according to the newly introduced structure

LakshmiKumar23 committed:
Merge branch 'develop' into swbs/lbp_fixes_pr10

Commit:
Merge branch 'swbs/lbp_fixes_pr10' into swbs/lbp_readers_pr11_merged

Commit:
Review comments from PR10 to PR11

Commit:
PR comments

Commit:
Merge remote-tracking branch 'upstream/develop' into swbs/lbp_readers_pr11_merged

SundarRajan28 reviewed (changes requested):
Added review comments

Commit:
Add file source reader .h file changes

Commit:
First level of comments - LBP

Commit:
LBP comments - 2

Commit:
Fix issues

Commit:
caffe2 reader fix

fgladwin committed:
Fix LBP file source reader

fgladwin committed:
Add count items fix for all readers

fgladwin committed:
Fix caffe and caffe2 readers

Fix shuffle in caffe
Init variable in caffe2

fgladwin committed:
Minor changes

fgladwin committed:
Minor changes

fgladwin committed:
Minor changes

fgladwin committed:
Remove get dataset size function

fgladwin committed:
Minor changes

fgladwin committed:
Merge branch 'develop' of https://github.com/ROCmSoftwarePlatform/rocAL into swbs/lbp_readers_pr11_merged

LakshmiKumar23 committed:
Merge branch 'develop' into swbs/lbp_readers_pr11_merged

fiona-gladwin committed:
Merge branch 'develop' into swbs/lbp_readers_pr11_merged

kiritigowda committed:
Merge branch 'develop' into swbs/lbp_readers_pr11_merged

LakshmiKumar23 committed:
Merge branch 'develop' into swbs/lbp_readers_pr11_merged

SundarRajan28 committed:
Fixing issues with ext src reader example

fgladwin committed:
Move all common members to Reader base class

fgladwin committed:
Rename last batch info to sharding info in file source reader

fgladwin committed:
Modify count items function

fgladwin committed:
Move pad file names to common Reader class

fgladwin committed:
Introduce Image reader file

fgladwin committed:
Minor changes

fgladwin committed:
Add comments and minor changes

fgladwin committed:
Merge branch 'fg/lbp_readers_pr11_combined' into swbs/lbp_readers_pr11_merged

dependabot[bot] committed:
Bump rocm-docs-core[api_reference] from 1.8.2 to 1.8.3 in /docs/sphinx (#225)

Bumps [rocm-docs-core[api_reference]](https://github.com/ROCm/rocm-docs-core) from 1.8.2 to 1.8.3.
- [Release notes](https://github.com/ROCm/rocm-docs-core/releases)
- [Changelog](https://github.com/ROCm/rocm-docs-core/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/ROCm/rocm-docs-core/compare/v1.8.2...v1.8.3)

---
updated-dependencies:
- dependency-name: rocm-docs-core[api_reference]
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

fgladwin committed:
Introduce constructor for sharding info

Initialize all members with a single parameterized constructor
Add pybind support for the same

fgladwin committed:
Update Changelog

fgladwin committed:
Merge branch 'develop' of https://github.com/ROCmSoftwarePlatform/rocAL into swbs/lbp_readers_pr11_merged
